idaes/generic_models/properties/core/pure/ConstantProperties.py

```diff
@@ -0,0 +1,133 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Method to set constant pure component properties:
+
+All parameter indicies and units based on conventions used by the source
+"""
+from pyomo.environ import log, Var, units as pyunits
+
+from idaes.core.util.misc import set_param_from_config
+
+
+# -----------------------------------------------------------------------------
+# Heat capacities, enthalpies and entropies
+class Constant(object):
+    
+    class cp_mol_liq_comp(object):
+
+        @staticmethod
+        def build_parameters(cobj):
+            cobj.cp_mol_liq_comp_coeff = Var(
+                doc="Parameter for liquid phase molar heat capacity",
+                units=pyunits.J*pyunits.kmol**-1*pyunits.K**-1)
```

Comment:
Should we define the parameter in terms of the base units, rather than setting the to SI? Other modules have defined units from the source text, but this one is a lot more general.